### PR TITLE
Arabic Translation Added

### PR DESCRIPTION
**Pull request by yousbot**

### CODE REVIEW

**Comment by CoralineAda** [comment]
Thanks, putting out a call for native Arabic speakers to review the translation now.

File: version/1/4/ar/code_of_conduct.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+---
+المدى#
+
+هذا الميثاق السلوكي يطبق في المجالين، في فضاءآت المشاريع و في الفضاءآت العمومية عندما يقدم فرد ما مشروعه أو مجموعته. أمثلة تقديمات المشاريع أو المجموعات تشمل إستعمال بريدا إلكترونيا رسميا، النشر من خلال صفحة رسمية على الفايسبوك، أو التصرف كممثل معين في نشاط داخل أو خارج الإنترنيت.
+
+---
+تنفيذات#
+
+كل تصرفات فضة، تحرش، أو سلوكات أخرى غير مقبولة يمكن التبليغ عنها بالإتصال بفريق المشروع في "أضف البريد الإلكتروني". سيتم المراجعة و التحري عن كل الشكاوي، و كنتيجة ستأخد الإجراءآت اللازمة حسب الظروف. فريق المشروع مجبر أن يحافظ على سرية المراسل او المشتكي. تفاصيل أخرى عن سياسات تنفيذية محددة قد تنشر مفرقة لاحقا.
```
Comment (RakanNimer) [inline version/1/4/ar/code_of_conduct.md]: "أضف البريد الإلكتروني" should be replaced with the email to support@github.com ? 
It means : "Add email"

**Comment by CoralineAda** [comment]
That’s placeholder text. The project maintainer needs to insert their own email address.

File: version/1/4/ar/code_of_conduct.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+المشرفون عن المشروع الذين لا يحترمون أو لا ينفذون الميثاق السلوكي بكل إيمان و صدق قد يواجهون صرف مؤقت أو دائم كما هو محدد من طرف أعضاء الرئاسة للمشروع.
+
+---
+تنسيب#
+
+هذا الميثاق السلوكي مأخود من " " ، النسخة 1.4 في
```
Comment (RakanNimer) [inline version/1/4/ar/code_of_conduct.md]: Here the source of the covenant is not mentioned, I think  من " " should be replaced with من ${sourceOfTheConvenant} version 1.4 etc..

File: version/1/4/ar/code_of_conduct.md (hunk 1 above)
Comment (Bashar) [inline version/1/4/ar/code_of_conduct.md]: @CoralineAda the changes are A LOT to be just commented here. especially RTL bullets tehy are all flipped

I'll create a newer PR for each file

I finished the first file

P.S: I'm an one of the main translators of Django and many other packages
https://www.transifex.com/user/profile/KuwaitNET/

**Comment by CoralineAda** [comment]
Is this ready to merge?

**Pull request closed without merging**